guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>We will use the same <code>clone</code> method on the target organization to clone this site. However, for a Hub Premium organization, your <code>clone</code> renders an <code>initiative</code> with the site included. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>